pkg/ottl/e2e/e2e_test.go

```diff
@@ -75,6 +126,14 @@ func Test_e2e_editors(t *testing.T) {
 			statement: `merge_maps(attributes, attributes["foo"], "insert")`,
 			want: func(tCtx ottllog.TransformContext) {
 				tCtx.GetLogRecord().Attributes().PutStr("bar", "pass")
+				tCtx.GetLogRecord().Attributes().PutStr("bar", "pass")
+				tCtx.GetLogRecord().Attributes().PutStr("bar", "pass")
+				tCtx.GetLogRecord().Attributes().PutStr("bar", "pass")
```

Comment:
```suggestion
```
Repeated calls to `PutStr` shouldn't add extra items into the map, right? I thought it overwrites the value if the key is already present.

Reply:
That was a bad copy and paste. Removed